Add legacy support to function

dshkol opened the pull request:
for `census_vectors`, which was previously removed in last commit. Also fix some typos in documentation for `label_vectors`.

mountainMath commented:
The vignettes fail with this, I think we need to export the `census_vectors` function so that e.g. the barograph vignette will run through. We should probably also change the vignettes to use the new function, but we still should export this for a while so that outside code that uses this does not break.

dshkol commented:
Added export so should be fixed now. I checked with legacy use of `census_vectors` in the bargraph vignette, and it works, but then I switched to `label_vectors` to use current naming.
